packages/bot-web-ui/src/components/__tests__/bot-stopped.spec.tsx

```diff
@@ -82,4 +82,26 @@ describe('<BotStopped />', () => {
             expect(global.location.reload).toHaveBeenCalled();
         });
     });
+
+    it('should render onClickClose on press of enter keydown', async () => {
+        const mockEventListener = jest.fn();
```

Comment:
`const mockEventListener = jest.fn();` you can move to the upper scope (before part it() ) to avoid duplication
```suggestion
```

Reply:
Alright @maryia-matskevich-deriv